openfe/setup/ligand_network_planning.py

```diff
@@ -205,6 +205,7 @@ def generate_minimal_spanning_network(
     mappers: Union[AtomMapper, Iterable[AtomMapper]],
     scorer: Callable[[LigandAtomMapping], float],
     progress: Union[bool, Callable[[Iterable], Iterable]] = True,
+    n_processes: int = 1,
```

Comment:
Needs docstring addition.

Reply:
my suggestion would be:
'
n_processes : int  
  provide a number of cpu processes to modulate the cpu-parallelization of the netwokplanning algorithm
'